scripts/selinux/README.md

```diff
@@ -36,10 +50,61 @@ sudo restorecon -R /etc/nginx-agent
 ```
 
 ## Updating existing policy
-Copy the folder `scripts/selinux` over to your rhel server.
-Then run the following command to update the policy:
+Check for errors by using the `ausearch` command:
+```
+sudo ausearch -m AVC,USER_AVC,SELINUX_ERR,USER_SELINUX_ERR --raw -se nginx_agent -ts recent
+```
+Generate new rule based on the errors by using `audit2allow`:
+```
+sudo ausearch -m AVC,USER_AVC,SELINUX_ERR,USER_SELINUX_ERR --raw -se nms -ts recent | audit2allow
+```
+
+Copy the file `scripts/selinux/nginx_agent.te` to a centos 7 machine and update it with the output from the `audit2allow` command.
```

Comment:
Is there a way to make it more clear that we are not saying to only run the audit2allow command on centos 7 and then copy the file. But to run the audit2allow command on 8 & 9 etc copy the output of that to the nginx_agent.te file and then copy that to centos 7  to build ?

Reply:
yeah let me add a note there to make it more clear